examples/artemis-transaction-consumer/artemis_transaction_consumer.bal

```diff
@@ -0,0 +1,22 @@
+import ballerina/artemis;
+import ballerina/io;
+
+public function main() returns error? {
+    // Create the Artemis listener with `autoCommitAcks` set to `false`.
+    // In this case the message acknowledgements will be committed only
+    // if the transaction block completes successfully.
+    artemis:Listener lis = new artemis:Listener({host:"localhost", port:61616});
+    // Create a `Consumer` object to make blocking receive calls
```

Comment:
```suggestion
    // Create a `Consumer` object to block receiving calls.
```
Is that what needs to be meant? The idea is not clear.

Reply:
Meaning of blocking receive calls is "synchronous receive calls". I have modified the comments